File: packages/expo-analytics-segment/android/src/main/java/expo/modules/analytics/segment/SegmentPackage.java
package expo.modules.analytics.segment;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import org.unimodules.core.BasePackage;
import org.unimodules.core.ExportedModule;

public class SegmentPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new SegmentModule(context));
  }
}

File: packages/expo-analytics-segment/android/src/main/java/expo/modules/analytics/segment/SegmentPackage.kt
package expo.modules.analytics.segment

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class SegmentPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(SegmentModule(context))
  }
}
